.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches: master
 
 env:
-  NODE_VERSION: 12.x
+  NODE_VERSION: 18.x
```

Comment:
We should keep testing on Node.js 12 until we can upgrade the version of Node.js used to run this tool on our server.

Reply:
well, that breaks using modern javascript async/await ect :/
What is needed to upgrade node.js on the www server?

Reply:
Current server is running Ubuntu 16.04 and using nodesource to install Node.js. I'm not sure whether Nodesource provide later versions of Node.js on Ubuntu 16.04. I'd like to update the OS on the server but it's risky given how dependent we are on that droplet at the moment.

Reply:
so we would either need to adjust this to run on node 12 or upgrade node without using nodesource, since upgrading the OS is not trivial
I prefer the second option, is that possible?

Reply:
I think Node.js 18 is a non-starter on Ubuntu 16.04 due to glibc versions. Node.js 16 is a possibility -- there may even be a Nodesource distribution for it (I'd suggest testing in a separate Ubuntu 16.04 system first).

Reply:
would testing on an ubuntu 16.04 docker container be sufficient? that's what I have in handy

Reply:
Based on the fact that https://deb.nodesource.com/node_16.x/dists/xenial/Release returns something, I think v16.x can be installed using the NodeSource distribution.

Reply:
I was able to install node16 on ubuntu 16.04 using nodesource. let me check these changes work on node 16

Reply:
tests have passed. @richardlau should I go ahead and update www nodejs version?

Reply:
@MoLow Yes (and keep an eye on tomorrow's nightly to check the indexer still runs ok). If you could also update https://github.com/nodejs/build/blob/b8b63ade1e76d8ccf669335ee384ed2ee34a3abe/ansible/www-standalone/tasks/base.yaml#L1-L3 (which says Node.js 8 but we are definitely running with 12 on the server) that would be great.